runtime/lua/vim/_defaults.lua

```diff
@@ -213,20 +213,50 @@ end
 --- Default menus
 do
   --- Right click popup menu
-  -- TODO VimScript, no l10n
-  vim.cmd([[
-    vnoremenu PopUp.Cut                     "+x
-    vnoremenu PopUp.Copy                    "+y
-    anoremenu PopUp.Paste                   "+gP
-    vnoremenu PopUp.Paste                   "+P
-    vnoremenu PopUp.Delete                  "_x
-    nnoremenu PopUp.Select\ All             ggVG
-    vnoremenu PopUp.Select\ All             gg0oG$
-    inoremenu PopUp.Select\ All             <C-Home><C-O>VG
-    anoremenu PopUp.Inspect                 <Cmd>Inspect<CR>
-    anoremenu PopUp.-1-                     <Nop>
-    anoremenu PopUp.How-to\ disable\ mouse  <Cmd>help disable-mouse<CR>
-  ]])
+  local function def_menu(ctx)
+    vim.cmd([[
+      vnoremenu PopUp.Cut                     "+x
+      vnoremenu PopUp.Copy                    "+y
+      anoremenu PopUp.Paste                   "+gP
+      vnoremenu PopUp.Paste                   "+P
+      vnoremenu PopUp.Delete                  "_x
+      nnoremenu PopUp.Select\ All             ggVG
+      vnoremenu PopUp.Select\ All             gg0oG$
+      inoremenu PopUp.Select\ All             <C-Home><C-O>VG
+      anoremenu PopUp.-1-                     <Nop>
+      anoremenu PopUp.Inspect                 <Cmd>Inspect<CR>
+      anoremenu PopUp.Go\ to\ definition      <Cmd>lua vim.lsp.buf.definition()<CR>
+      amenu     PopUp.Open\ in\ web\ browser  gx
+      anoremenu PopUp.-2-                     <Nop>
+      anoremenu PopUp.How-to\ disable\ mouse  <Cmd>help disable-mouse<CR>
+    ]])
+
+    vim.cmd([[
+      amenu disable PopUp.Open\ in\ web\ browser
+      amenu disable PopUp.Go\ to\ definition
```

Comment:
`:menu disable` currently removes the item from the popupmenu. Would be nice if there were a way to show the item but have it "greyed out".